### PR TITLE
ci: update script to publish from ci

### PR DESCRIPTION
**Pull request by haxxmaxx**
Based on sn-tables ci script. Publishes if on master, the tag and version in package.json has been bumped

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,4 +75,96 @@ jobs:
...
           path: coverage/junit
-      - save_cache: *save_yarn_cache
+      - persist_to_workspace:
+          root: ~/action-button
+          paths:
+            - dist
+            - package.json
+            - core
+
+  # publish-dev:
```
Comment (haxxmaxx) [inline .circleci/config.yml:85]: I've never used this, but we have it for several charts. I guess it can be removed?

Comment (niekvanstaveren) [reply .circleci/config.yml:85]: I guess you can use it to test your extension? Is it commented in all other charts as well? In that case better to remove I think

**Review by niekvanstaveren — COMMENTED** [review]
Looks good. Should check a release after this probably

**Comment by niekvanstaveren** [comment]
Probably you want to update the README as well

File: .circleci/config.yml (hunk 1 above)
Comment (haxxmaxx) [reply .circleci/config.yml:85]: No I commented it now

**Comment by haxxmaxx** [comment]
> Probably you want to update the README as well

Ah yes, will fix that

File: .github/CONTRIBUTION.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+## Developing
+
+1. Install with `yarn`
+1. Transpile code: `yarn build` (or `yarn build:watch`)
+1. Link to sense-client (locally on your computer):
```
Comment (haxxmaxx) [inline .github/CONTRIBUTION.md:5]: Not sure we want this step here, you need access to sense-client. @niekvanstaveren

Comment (niekvanstaveren) [reply .github/CONTRIBUTION.md:5]: No we should not have this in the open source repo. Probably only mention yarn start here.

Comment (haxxmaxx) [reply .github/CONTRIBUTION.md]: done

**Review by niekvanstaveren — APPROVED** [review]

**Pull request merged**